xds/src/main/java/io/grpc/xds/EdsLoadBalancer.java

```diff
@@ -114,6 +117,9 @@ public void handleResolvedAddresses(ResolvedAddresses resolvedAddresses) {
       return;
     }
     XdsConfig newXdsConfig = (XdsConfig) lbConfig;
+    if (Objects.equals(newXdsConfig, xdsConfig)) {
+      return;
```

Comment:
(If there is attributes or addresses change in `resolvedAddresses`) we still need call `switchingLoadBalancer.handleResolvedAddresses(resolvedAddresses)`
before return.

Reply:
Sure, removed this block. The following two blocks will only happen when specific fields (i.e., `lrsServerName`, `edsServiceName`) in lb config changes. ResolvedAddresses is always delegated to `switchLoadBalancer.handleResolvedAddresses(...)`.